components/eam/src/physics/crm/samxx/samxx_const.h

```diff
@@ -231,11 +229,38 @@ real constexpr nzeros = 3.e6;   // Intersept coeff. for snow
 real constexpr nzerog = 4.e6;   // Intersept coeff. for graupel
 real constexpr qp_threshold = 1.e-8; // minimal rain/snow water content
 
-real constexpr qcw0 = 1.e-3;
-real constexpr qci0 = 1.e-4;
 real constexpr alphaelq = 1.e-3;
 real constexpr betaelq = 1.e-3;
 
+//------------------------------------------------------------------------------
```

Comment:
All these fixes should be in master now.

Reply:
ah, this was an accident due to how I created this branch from the previous PR branch. I don't think it will cause any conflicts.